src/main/scala/ai/privado/passes/SQLPropertyPass.scala

```diff
@@ -0,0 +1,100 @@
+package ai.privado.passes
+
+import ai.privado.cache.RuleCache
+import ai.privado.languageEngine.java.language.NodeStarters
+import ai.privado.model.sql.{SQLColumn, SQLQuery}
+import ai.privado.tagger.PrivadoParallelCpgPass
+import ai.privado.utility.SQLParser
+import io.shiftleft.codepropertygraph.generated.{Cpg, EdgeTypes}
+import io.shiftleft.codepropertygraph.generated.nodes.{
+  File,
+  JavaProperty,
+  NewFile,
+  NewSqlColumnNode,
+  NewSqlQueryNode,
+  NewSqlTableNode
+}
+import org.slf4j.LoggerFactory
+
+class SQLPropertyPass(cpg: Cpg, projectRoot: String, ruleCache: RuleCache)
+    extends PrivadoParallelCpgPass[JavaProperty](cpg) {
+
+  val logger = LoggerFactory.getLogger(getClass)
+
+  override def generateParts(): Array[_ <: AnyRef] =
+    cpg.property.l.filter(prop => prop.name.matches("(?i)query")).toArray
+
+  override def runOnPart(builder: DiffGraphBuilder, property: JavaProperty): Unit = {
+    addSqlNodes(builder, property)
+  }
+
+  private def addSqlNodes(builder: SQLPropertyPass.this.DiffGraphBuilder, property: JavaProperty): Unit = {
+
+    val query        = property.value
+    val lineNumber   = property.lineNumber.getOrElse(-1).asInstanceOf[Int]
+    val columnNumber = property.columnNumber.getOrElse(-1).asInstanceOf[Int]
+
+    println(query)
+
+    try {
+      SQLParser.parseSqlQuery(query) match {
+        case Some(parsedQueryList) =>
+          parsedQueryList.zipWithIndex.foreach { case (parsedQueryItem: SQLQuery, queryOrder) =>
+            buildAndReturnIndividualQueryNode(
+              builder,
+              property.sourceFileOut.head,
+              parsedQueryItem,
+              query,
+              property.lineNumber.getOrElse(-1).asInstanceOf[Int],
+              queryOrder
+            )
+          }
+        case None =>
+          println("failed")
+          logger.debug("Failed to parse")
```

Comment:
remove println and have some description message in debug to understand what has failed